### DISCUSSION TITLE
Clarification on drug file

### DISCUSSION THREAD

**Discussion by sklukasatphs** [CMSgov/hospital-price-transparency#139] [Q&A] [answered]
Does it matter which code type we use in the MRF with regard to drugs - HCPCS code or NDC code?  HCPCS codes typically have multiple NDC codes with multiple dosages, but we charge for and are reimbursed on the HCPCS code dosage.  As has been mentioned by other folks, the standard gross charge on the chargemaster is $0 but a pricing algorithm is applied when administered and charged.  Does CMS have a preference for HCPCS code or NDC code?

**Comment by RidenourK** [comment, marked as answer]
@sklukasatphs. Thank you for your question. 

In the CY2024 OPPS/ASC Final Rule, we agreed with commenters that billing codes bring necessary context to the standard charges established by hospitals. We additionally agreed that more than one code may be necessary to establish that context (for example, a HCPCS code plus a revenue center code may be needed for describing a gross charge)….We also indicated we did not believe it would be in the public's best interest to limit the types of codes hospitals can use to describe the standard charges they establish because it may increase the “N/As”. Additionally, we agreed that gross charges that are established by the hospital for itemized items and services are often associated with CPT and HCPCS codes in the hospital's chargemaster, whereas it may be more appropriate to contextualize the payer-specific negotiated charges that hospitals have established with third party payers with DRGs, APCs, or other types of payer codes. We further recognized that payer-specific negotiated charges may depend on a variety of factors, which may make it challenging to display as a single dollar amount. In such cases, a hospital may have established payer-specific negotiated charges that can only be expressed as an algorithm. When this occurs the hospital if required to describe the algorithm that applies and calculate and encode an estimated allowed amount… We clarified to the extent a hospital uses one or more codes to bill/account for items and services for which the hospital has established a standard charge, the hospital must indicate these in its MRF. Please refer to the discussion beginning at [88 FR 82104](https://www.federalregister.gov/d/2023-24293/p-5018).
